backend/tracim_backend/models/setup_models.py

```diff
@@ -122,9 +122,6 @@ def init_models(configurator: Configurator, app_config: "CFG") -> None:
     # use pyramid_tm to hook the transaction lifecycle to the request
     configurator.include("pyramid_tm")
 
-    # use pyramid_retry to retry a request when transient exceptions occur
-    configurator.include("pyramid_retry")
```

Comment:
as pyramid_retry as some caveat and we have no clear reason to use it, i do think it's better to remove it. 
https://docs.pylonsproject.org/projects/pyramid-retry/en/latest/#caveats

The idea behind pyramid_retry is that is some specific case the transaction manager (with sqlalchemy) is able to know that an error can be fixed by retrying doing the request, so they may be some hypothetical case where this change something but it's something that in never encounter and i do think it's better to know these issue instead of hidding them behind some weird magic we don't completely understood.